README.md

```diff
@@ -1,114 +1,131 @@
-
-# 🌷 Rails Blog  🌷
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
 
```

Comment:
- [ ] Kindly go through the project requirements and be sure to follow the guidelines. You have eliminated the N+1 query problem for controller actions like the pos index, but I cannot find any single file related to the integration tests for the project using Capybara. I cannot find any integration test file in this PR or in your repository file structure on this branch. Kindly go through the guidelines and add your tests.

  The tests are conventionally placed in the `spec/integrations` or `spec/features` folder. Kindly add all the tests listed in the guidelines and ensure they pass. If you are struggling with capybara you can visit [this cheat sheet](https://gist.github.com/zhengjia/428105)